doc/whats_new/v1.3.rst

```diff
@@ -155,6 +155,9 @@ Changelog
 - |Fix| :func:`metric.manhattan_distances` now supports readonly sparse datasets.
   :pr:`25432` by :user:`Julien Jerphanion <jjerphan>`.
 
+- |API| adding :func:`metric.paired_haversine_distances` to compute pairwise haversine distances.
+  :pr:`22705` by :user:`Jona Sassenhagen <jona-sassenhagen>` and :user:`Charles Patel <acharles7>`.
```

Comment:
```suggestion
- |API| :func:`metric.paired_haversine_distances` has been introduced to compute
  the haversine distance between pairs of vectors.
  :pr:`22705` by :user:`Jona Sassenhagen <jona-sassenhagen>` and :user:`Charles Patel <acharles7>`.
```

Reply:
I'm not sure what you mean? The main point of this PR is to add missing functionality, as discussed in https://github.com/scikit-learn/scikit-learn/issues/17212 Or what am I missing?